### DISCUSSION TITLE
Laravel 11 Release Date

### DISCUSSION THREAD

**Discussion by melokullc** [laravel/framework#50195] [Q&A] [answered]
Apologies if this information is or was posted somewhere official. I've referenced the Laravel docs and it still states a released date of Q1, and a few searches state Feb 6th.

But does anyone know 
A) an exact release date and/or 
B) some place better to reference official notes/releases aside from the docs.

Thanks

**Comment by henzeb** [comment, marked as answer]
Taylor was talking at Laracon EU about the beginning of March I believe...
